### ISSUE TITLE
Changing the NumberBox value with Arrow Keys makes the UI unresponsible for the mouse.

### ISSUE THREAD

**Issue by tpoint75** [microsoft/microsoft-ui-xaml#10223]
### Describe the bug

Changing the NumberBox value with Arrow Keys makes the UI unresponsible for the mouse.

### Steps to reproduce the bug

Use a NumberBox in WinApp SDK 1.6 in winrt/c++.
Click inside the Textbox and use Arrow Up Key to change the value.
Now hover with the mouse over other controls while keeping the Arrow Up Key pressed.
After a while the ui gets unresponsible for the mouse and you can't klick anything on the UI until enter is clicked,
to update the changed NumberBox value.

### NuGet package version

Windows App SDK 1.6 Preview 2: 1.6.240821007-preview2

### Packaging type

Unpackaged

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

### IDE

Visual Studio 2022

### Additional context

I cant remember to have seen this in WinAppSDK 1.5.

**Comment by tpoint75** [comment]
The updated WinUI Gallery shows the same behavior!

**Comment by ranjeshj** [comment]
I can repro as well. Looks like mouse and keyboard input need to be happening at the same time to repro.

**Comment by tpoint75** [comment]
What is the duplicate? Anw what do you mean with "not planned"? Leave the Bug as it is? Please try to improve your communication!

**Comment by codendone** [comment]
@lolametro Yes, this another script closure from the internal issue getting dup'ed to a non-public issue. I've asked the engineer to reactivate and/or clarify status.

**Comment by iablaauw-MS** [comment]
Appologies, this was an accidental closure. I'm reactivating this - it is being actively investigated.

**Comment by iablaauw-MS** [comment]
This is a duplicate of other similar issues related to arrow keys & mouse input at the same time.

This has been fixed in 1.6.7 and 1.7.1! Let us know if you run into it again.
